### DISCUSSION TITLE
How do I make mitmproxy ignore its config file?

### DISCUSSION THREAD

**Discussion by tddschn** [mitmproxy/mitmproxy#5350] [Ideas]
Sometimes I want to use `mitmproxy` without the configured options set in `~/.mitmproxy/config.yaml`.

Many command line tools like `vim` and `zsh` provide a flag to ignore config file and start fresh, how do I do the same with `mitmproxy`?

**Comment by mhils** [comment]
We don't have an option for this at the moment. We maybe want to add a `--config` argument or something along those lines at some point, but that does not exist yet. :)

**Comment by phaseOne** [comment]
[docker-compose does something like this](https://docs.docker.com/compose/reference/#use--f-to-specify-name-and-path-of-one-or-more-compose-files). You can use one or more `-f` options to specify the paths to `docker-compose.yml` files, and then the program merges the YAML files.
> When you supply multiple files, Compose combines them into a single configuration. Compose builds the configuration in the order you supply the files. Subsequent files override and add to their predecessors.
